Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -6307,6 +6307,15 @@ OverscrollBehavior LocalFrameView::verticalOverscrollBehavior()  const
     return OverscrollBehavior::Auto;
 }
 
+ScrollbarGutter LocalFrameView::scrollbarGutterStyle()  const
+{
+    auto* document = m_frame->document();
+    auto scrollingObject = document && document->documentElement() ? document->documentElement()->renderer() : nullptr;
+    if (scrollingObject && renderView())
```

Comment:
The `renderView` check is surprising enough that it needs a comment I think.

Reply:
I'll be honest I don't actually know what that checks for. This code is a copy of scrollbarWidthStyle which was a copy of verticalOverscrollBehavior.

Reply:
Will compile locally without it for both gutter and width and see if it breaks anything

Reply:
Doesn't seem to be important will remove it. Will also make a follow up to see if its needed for scrollbar width or the overscrollBehavior functions

Reply:
https://bugs.webkit.org/show_bug.cgi?id=259018